src/validator.rs

```diff
@@ -1548,6 +1549,56 @@ impl<'a> Validator<'a> {
         }
     }
 
+    /// Expect the block to contain any number of `key = value` fields
+    /// where each key is a unique Item of type itype.
+    /// Run the closure `f(key, vd)` for every matching block.
+    #[cfg(feature = "vic3")]
```

Comment:
It's okay to do `#[allow(dead_code)]` instead. That way you don't get the warning about unused import of TigerHashSet. (It's because it is unused when not compiling for vic3)